src/feature.py

```diff
@@ -0,0 +1,169 @@
+from chem_lex.ChemlabTokens import unit_prefix, supported_units
+#import
+from element import Element
+def convertTo(el, num, Unit, newUnit):
+    ### need to redo with the parser
+
+    if(Unit in unit_prefix and Unit in unit_prefix):
+
+        result = convertPrefix(num,Unit,newUnit)
+        print("Result:    " + str(result)+" "+ newUnit)
+    elif (Unit in supported_units and Unit in supported_units):
+
+        result = convertUnit(el, num,Unit,newUnit)
+        print("Result" + str(result)+" "+ newUnit)
+    else:
+        raise TypeError("Invalid Units is :  " + str(type(Unit)) )
+    return result
+
+def convertWeight(el ,num, Unit, newUnit):
+    ### Converts weight related to an element
+    if not isinstance(el, Element):
+        raise TypeError("Not an element. Cannot perform operation")
+    else:
+        amu = el.dictionary['atomic_weight']
+        if(Unit=='g'):
+            if(newUnit=='mol'):
+                result= num/amu
+            elif(newUnit=='atoms'):
+                mol = num/amu
+                result = mol * (6.02*(10**23))
+            else:
+                raise TypeError("It's not possible to make that conversion")
+        elif(Unit == 'mol'):
+            if newUnit=='atoms':
+                result = result = num*(6.02*(10**23))
+            if newUnit=='g':
+                result = num*amu
+            else:
+                raise TypeError("It's not possible to make that conversion")
+        elif(Unit=='atoms'):
+            if newUnit=='mol':
+                result = num/(6.02*(10**23))
+            if newUnit=='g':
+                mol = num/(6.02*(10**23))
+                result = mol * amu
+            else:
+                raise TypeError("It's not possible to make that conversion")
+        else:
+            raise TypeError("It's not possible to make that conversion")
+
+    return result
+
+
+pre = {'G': 10**9,  # GIGA
```

Comment:
Im not sure how the prefixes are being used cause Im not sure how they are passed in. We need to talk about this a bit